addons/pkg-npm/jaxrs/src/main/java/org/commonjava/indy/pkg/npm/jaxrs/NPMContentAccessHandler.java

```diff
@@ -363,18 +362,20 @@ else if ( item.isDirectory() && StoreType.remote != st )
                     {
                         // for remote retrieve, when content has downloaded and cached in the mapping path,
                         // the item here will be a directory, so reassign the path and item as the mapping one
-                        if ( item.isDirectory() && StoreType.remote == st )
-                        {
-                            path = PathUtils.storagePath( path, eventMetadata );
-                            origItem = item;
-                            item = contentController.get( sk, path, eventMetadata );
-                        }
 
-                        if ( item == null )
-                        {
-                            logger.error( "Retrieval of actual storage path: {} FAILED!", path );
-                            throwError( ApplicationStatus.SERVER_ERROR, new NullPointerException( path ), "Retrieval of mapped file from storage failed." );
-                        }
+                        // Note: as STORAGE_PATH is not used, these code is also useless now.
+//                        if ( item.isDirectory() && StoreType.remote == st )
+//                        {
+//                            path = PathUtils.storagePath( path, eventMetadata );
```

Comment:
Do we still need STORAGE_PATH constant and PathUtils.storagePath() at all now?

Reply:
Actually not needed anymore. I'll try to remove them.